cvat/apps/dataset_manager/bindings.py

```diff
@@ -268,6 +268,8 @@ def get_frame(idx):
         for shape in sorted(anno_manager.to_shapes(self._db_task.data.size),
                 key=lambda shape: shape.get("z_order", 0)):
             if 'track_id' in shape:
+                if shape['outside']:
```

Comment:
@zhiltsov-max , will the code work correctly for the case when a user wants to dump annotations in CVAT for video format? Because the format has to include outside frames with explicit `outside=1`. Could you please remind me how the case will be handled?

Reply:
This function is not used in that format.